roles/basic-prog-pkgs/vars/main.yml

```
@@ -1,2 +1,18 @@
 ---
-# vars file for basic-prog-pkgs
\ No newline at end of file
+# vars file for basic-prog-pkgs
+basic_prog_pkgs_intro_development:
+ -artha
```

Can you add a space after the `-`?

```suggestion
 - artha
```